### DISCUSSION TITLE
scrcpy second screen

### DISCUSSION THREAD

**Discussion by CorruptionVR** [yume-chan/ya-webadb#418] [Q&A] [answered]
Is it possible to use scrcpy an s second screen to extend my device's screen? Or something like Samsung DeX?

**Comment by yume-chan** [comment, marked as answer]
It has the same capabilities as the original Scrcpy. You can manually add a secondary screen (see https://github.com/Genymobile/scrcpy/issues/1887#issuecomment-820309286) and use my client to mirror it.

![image](https://user-images.githubusercontent.com/1330321/168006602-5eb6366b-408b-4ef8-99f6-5ab16f10f506.png)

![image](https://user-images.githubusercontent.com/1330321/168006506-ef7b41fc-2815-4804-bb15-2b7ed776b290.png)

**Reply by yume-chan** [reply]
However, according to https://github.com/Genymobile/scrcpy/issues/898, Samsung Dex mode needs a real HDMI connection to work.

**Comment by CorruptionVR** [comment]
it works. now I just need to know how to get the simulated display off my first screen. do you know how to do that? is it even possible? because it turned on `simulate secondary displays`, and the new display goes away when I put it back to none.
